src/contracts/Avalanche-Parameter-Rec-21-02-2023/Avalance-Parameter-Rec-21-02-2023.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
```

Comment:
```suggestion
pragma solidity ^0.8.16;
```
needs to be higher than 16, because of constant assignments `LINKe = AaveV3AvalancheAssets.LINKe_UNDERLYING;`